### PR TITLE
fix cli

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit e8de7629c818cc5eb8e45b48895f0831f1aad136

https://app.netlify.com/sites/using-glamor/deploys/59fa237da6188f2434f85924

File: packages/gatsby-cli/gatsby.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
```
Comment (KyleAMathews) [inline packages/gatsby-cli/gatsby.js]: was this supposed to have been added back?

Comment (jquense) [reply packages/gatsby-cli/gatsby.js]: yeah, this is the entry, i was trying to just replace the old bin.js with something with the right permissions

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit b2fa86d85aa78162ffdb5c4697d09b4e90c6044c

https://app.netlify.com/sites/using-glamor/deploys/59fa23b40b79b73c6a3d0c6f

File: packages/gatsby-cli/gatsby.js (hunk 1 above)
Comment (KyleAMathews) [reply packages/gatsby-cli/gatsby.js]: k, ready to merge and publish again?

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit e996ff163a07d69e6b57599bf0dfa14c108dff3f

https://app.netlify.com/sites/using-glamor/deploys/59fa246fdf995348712abece

File: packages/gatsby-cli/gatsby.js (hunk 1 above)
Comment (jquense) [reply packages/gatsby-cli/gatsby.js]: yup

**Comment by jquense** [comment]
ARG its still failing. I have no idea what is wrong here

**Comment by jquense** [comment]
the original error I mean

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 8e7c5d74006b1df5a561b04219059bc69034d0bb

https://app.netlify.com/sites/using-glamor/deploys/59fa2496a6188f2434f859fd

**Comment by KyleAMathews** [comment]
Isn't the problem (before this PR) just that node is failing to require gatsby-cli because the main is wrong now?

**Comment by jquense** [comment]
There was no bin file at all in the package, so it wasn't failing because of a permission because it didn't exist. it needs to exist tho

**Comment by jquense** [comment]
i do not understand this I have this _EXACT_ same setup on a work project with no problem

**Comment by jquense** [comment]
ARG THIS IS DUMB we don't even need the bin file for these installs its only for other people installing it

**Comment by KyleAMathews** [comment]
Well... let's merge this even if it's broken on Netlify so at least gatsby-cli works for people again. #sigh

**Pull request merged**